src/EditorFeatures/Core/InlineRename/InlineRenameSession.cs

```diff
@@ -588,7 +589,8 @@ private void QueueApplyReplacements()
         return (replacementInfo, mergeResult);
     }
 
-    private void ApplyReplacements(IInlineRenameReplacementInfo replacementInfo, LinkedFileMergeSessionResult mergeResult, CancellationToken cancellationToken)
+    private async Task ApplyReplacementsAsync(
+        IInlineRenameReplacementInfo replacementInfo, LinkedFileMergeSessionResult mergeResult, CancellationToken cancellationToken)
     {
         _threadingContext.ThrowIfNotOnUIThread();
```

Comment:
This should be changed to `await _threadingContext.JoinableTaskFactory.SwitchToMainThreadAsync` (which will be a NOP but will better fit expected patterns with the return type change)

Reply:
Will do.

Reply:
done.